### DISCUSSION TITLE
Scenario editor | Select ALL/NONE

### DISCUSSION THREAD

**Discussion by robin19972020** [OpenRCT2/OpenRCT2#15343] [Feature requests]
When creating a new scenario, you have to **manually** click every single ride/stall/etc.

I would really enjoy having a checkbox that selects all options for the scenario creator.
Saves me alot of time and is simply a quality of life feature to enjoy having around.

:)

**Comment by spacek531** [comment]
ok, what all options would you like?

**Comment by Snaid1** [comment]
given the new save format has expanded limits, I'd like this too! if there is somehow more than the limit number of attractions/scenery objects/etc, have it select them in order until the limit is hit and the scenario creator can modify from there.

**Comment by 733737** [comment]
If the creators want to limit this select all to only the ride tab or other tabs or limit it to exclude custom scenery I am okay with that if it would prevent crashes or make the creators less worried. They have mentioned concern with file sizes for testing or sharing parks.

**Comment by SpartanFrederic104** [comment]
Speaking of this, it would also be nice to allow selecting all objects from a pre-defined or custom-made object group. For instance:

- **Mythological**
  - Mythological Themeing
  - Mythological Entrance
  - Mosaic Footpath
  - Balustrade
  - Flying Saucers - Cyclops Ride
  - Car Ride - Pegasus Cars
  - Hyper-Twister - Cerberus Trains
  - Lay-down Roller Coaster - Harpies Trains
  - Vertical Drop Roller Coaster - Valkyries Trains
  - Twist - Neptune Ride
  - River Rafts - River Styx Boats
  - Food Stall - Neptune's Seafood Stall

**Comment by karst** [comment]
You can currently make theme "benches" for selecting your default items. Themes can select any object. 

An option to deselect all unused objects, that's outside the console, would definitely be a nice addition to the object selector, though.

**Comment by RockHardWarrior** [comment]
A select all feature would be incredibly helpful in creating original parks/scenarios, having to click "yes" to +2000 options is brutal.

**Comment by ZeeMaji** [comment]
This is available as a plugin: https://openrct2plugins.org/plugin/R_kgDON9bo4w/openrct2-auto-object-loader
